src/vivarium/framework/state_machine.py

```diff
@@ -83,7 +83,8 @@ def _groupby_new_state(
 
     """
     output_map = {o: i for i, o in enumerate(outputs)}
-    groups = pd.Series(index).groupby([output_map[d] for d in decisions])
+    # Pass grouper as Series to avoid FutureWarning about passing list of length 1
+    groups = pd.Series(index).groupby(pd.Series([output_map[d] for d in decisions]))
```

Comment:
The index for the series in the groupby call just gets ignored I'm assuming?

Reply:
The groupby documentation says "If a dict or Series is passed, the Series or dict VALUES will be used to determine the groups (the Series’ values are first aligned; see .align() method)".

Reply:
I'm curious to hear how you figured out this would work

Reply:
I was thinking of how I could pass data containing a single element without a list and without passing the element directly, so I looked up the groupby documentation to see what data structures I could use and thought a dictionary and Series were promising. I first tried a dictionary, but that created a different groupby object than the original. A Series reproduced the old behavior.